lntest/itest/lnd_test.go

```diff
@@ -8598,9 +8602,18 @@ func assertDLPExecuted(net *lntest.NetworkHarness, t *harnessTest,
 	carol *lntest.HarnessNode, carolStartingBalance int64,
 	dave *lntest.HarnessNode, daveStartingBalance int64) {
 
+	// We disabled auto-reconnect for some tests to avoid timing issues.
+	// To make sure the nodes are initiating DLP now, we have to manually
+	// re-connect them.
+	ctxb := context.Background()
+	err := net.ConnectNodes(ctxb, carol, dave)
+	if err != nil && !strings.Contains(err.Error(), "already connected") {
```

Comment:
If they're supposed to be disconnected before this point, wouldn't it be an error if they already were?

Reply:
The tests relied on the auto-reconnect feature which was wrong IMO as it depends on the timing of the node startup and the `--minbackoff` setting. Now with the min backoff set very high (which is equal to disabling reconnect), we have to do an explicit reconnect which is what we want.

Reply:
Gotcha, so now they should never reconnect unless done explicitly, but wouldn't ignoring the already connected error imply that they somehow did though?

Reply:
A bunch of tests use this function so I didn't want to break them in case they implemented a manual reconnect for whatever reason.